migrations/900000000000059_remove_empty_tables.up.sql

```diff
@@ -0,0 +1,45 @@
+BEGIN;
+-- code to drop empty tables in public schema as tables are now declared by their coresponding sync
+DO $$
+DECLARE empty_tables_sql TEXT;
+BEGIN    
+    WITH empty_table_candidates AS (
+        SELECT relname as table_name
+        FROM pg_class
+        JOIN pg_namespace ON (pg_class.relnamespace = pg_namespace.oid)
+        WHERE 
+            -- has not recorded pages however this can be wrong when there are just a few rows so we have to count rows in the next CTE
+            relpages = 0 
+            AND
+            -- only select tables
+            relam <> 0
+            AND 
+            pg_namespace.nspname = 'public'
+            AND 
+            relname NOT IN ('repos', 'schema_migrations', 'schema_migrations_history', 'sqlq_migrations')
```

Comment:
As discussed on our call the reason we use pg_class and pg_namespace is to be able to do a corse filter of repages = 0 so that we can only count rows on small or empty tables and make the task fast.